src/components/Tooltip/index.tsx

```diff
@@ -16,31 +16,36 @@ export interface TooltipProps {
   position?: 'top' | 'left' | 'right' | 'bottom';
 }
 
-const Tooltip = ({
-  trigger,
-  message,
-  position = 'bottom',
-  maxWidth = 240,
-}: TooltipProps): JSX.Element => (
-  <RadixTooltip.Provider>
-    <RadixTooltip.Root delayDuration={350}>
-      <RadixTooltip.Trigger asChild>
-        <span>{trigger}</span>
-      </RadixTooltip.Trigger>
-      <RadixTooltip.Content
-        side={position}
-        sideOffset={parseInt(FDS.SPACE_S, 10)}
-        asChild
-      >
-        <div
-          style={{ maxWidth: maxWidth ? `${maxWidth}px` : undefined }}
-          className="ease-in-out bgColor--charcoal inverted align--center padding--y--s padding--x elevation--2 rounded--all fontSize--s fontWeight--bold"
-        >
-          {message}
-        </div>
-      </RadixTooltip.Content>
-    </RadixTooltip.Root>
-  </RadixTooltip.Provider>
+const Tooltip = React.forwardRef<HTMLDivElement, TooltipProps>(
+  (
+    { trigger, message, position = 'bottom', maxWidth = 240 }: TooltipProps,
+    forwardedRef
+  ) => {
+    const refContent = forwardedRef || useRef<HTMLDivElement>(null);
+
+    return (
+      <RadixTooltip.Provider>
+        <RadixTooltip.Root delayDuration={350}>
+          <RadixTooltip.Trigger asChild>
+            <span ref={refContent}>{trigger}</span>
```

Comment:
i kind of feel that we agreed on forwarding the ref to the trigger and removing the span, didn't we?
i could be wrong.

Reply:
RadixTooltip.Trigger doesn't allow a ref prop on it

Reply:
gotcha, then I guess passing it to the `<span>` and using that `asChild` makes sense

Reply:
yeah idk if there's any other way to do it